### DISCUSSION TITLE
_TW and Gutemberg blocks

### DISCUSSION THREAD

**Discussion by Vince-ALIEN** [gregsullivan/_tw#116] [General]
Hello Greg,
I wanted to thank you for all this work on _TW. It's really a great base.

I'm trying to create a plugin with column blocks and a container. It seems to work correctly for the first time. 
Here is the link :https://github.com/Vince-ALIEN/tw-bloks.

If you (or anyone else) have a little time to tell me if I have made any beginner's mistakes and guide me to better integrate these blocks. I would also like to be able to add BlockVariationPicker, but that's still a mystery at the moment.
 THANKS .

**Comment by gregsullivan** [comment]
Hi Vincent—Thanks for posting this!

That repository appears to be private. If you make it public, I'll take a look!

**Reply by Vince-ALIEN** [reply]
Sorry...

**Reply by Vince-ALIEN** [reply]
It's done...

**Reply by gregsullivan** [reply]
Thanks for that! I'm out of the office today, but I'll check back in on this next week. Enjoy your weekend!

**Reply by gregsullivan** [reply]
Hi again, Vincent—Very sorry for the delay in getting back to you!

I took a look at your repository. Things are looking good as far as I can tell! These are minor notes, but some things I noticed:

* Depending on your workflow, you may not need to commit the `build` folder on GitHub. It makes for noisier commits if the built files are always being committed at the same time as changes to your source files. The argument for committing a build directory normally comes down the something like wanting to deploy directly from the repository without a build step. Could be worth considering, in any case!
* You could probably simplify your Tailwind-related scripts in `package.json` from this:

```json
"tailwindbuild": "tailwindcss -i ./src/input.css -o ./build-assets/index.css --postcss --minify",
"tailwindwatch": "tailwindcss -i ./src/input.css -o ./build-assets/index.css --watch --postcss",
"tailwind:watch": "npm run tailwind:build -- --watch",
```

to something like this:

```json
"tailwind:dev": "tailwindcss -i ./src/input.css -o ./build-assets/index.css --postcss",
"tailwind:watch": "npm run tailwind:dev -- --watch",
"tailwind:prod": "npm run tailwind:dev -- --minify",
```

* This isn't really related to anything I saw, but I really like the `10up/block-components` package, and you might find it useful at some point.

If you have specific questions, I'll do my best to answer them!

**Reply by Vince-ALIEN** [reply]
Hello Greg, I haven't forgotten you, excuse me for this very late response. I thank you for your response. I worked a lot to arrive at something that begins to resemble something. You can see the result in operation [here](https://dev.ufo-agency.com/)...

**Reply by gregsullivan** [reply]
Hi Vincent—That looks great! Thanks for sharing it. I'm always very happy to see what people have been able to build with _tw, and I'm happy my earlier response helped.

Best of luck with your site building in the future!
